packaging/docker/sidecar.py

```diff
@@ -336,6 +344,35 @@ def extract_desired_ip(cls, version, string):
             ip = f"[{ip}]"
         return ip
 
+    def read_node_zone_topology_label(hostname):
+        try:
+            k8s_config.load_incluster_config()
+        except k8s_config.ConfigException:
+            raise Exception("Could not configure kubernetes python client")
+
+        client=k8s_client.CoreV1Api()
+
+        # RBAC configuration may be required to access node resources.
+        # ref: https://github.com/kubernetes-client/python/blob/master/examples/in_cluster_config.py
+        api_response = client.read_node(hostname)
+
+        labels = api_response.metadata.labels
+
+        if not labels:
+            log.error("The node does not have any labels")
+
+        # failure-domain.beta is deprecated, with topology being the new value. This can vary between clusters/versions.
+        # https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints
+        if 'failure-domain.beta.kubernetes.io/zone' in labels:
```

Comment:
Should we move those constants into a dedicated string instead of duplicating the string? Do you know if the python client library has those constants defined? I know that the go client has some of them defined.

Reply:
apparently not :(